tidb-cloud/cli-reference.md

```diff
@@ -7,6 +7,10 @@ Summary: Provides an overview of TiDB Cloud CLI.
 
 TiDB Cloud CLI is a command line interface, which allows you to operate TiDB Cloud from your terminal with a few lines of commands. In the TiDB Cloud CLI, you can easily manage your TiDB Cloud clusters, import data to your clusters, and perform more operations.
 
+> **Note:**
```

Comment:
For some features such as deleting cluster,  it also work with dedicated cluster.